File: gang/week6/2016년.java
class Solution {
    public String solution(int a, int b) {
        String[] day = {"FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"};   // 금요일 시작이므로 금요일부터 요일 배열 생성
        int[] totalDay = {31,29,31,30,31,30,31,31,30,31,30,31}; // 각 월 별 최대 일잘 배열 생성
        int days = 0;

        for(int i = 0; i< a - 1; i++) {     // 찾으려는 달의 전 달까지 총 일수를 더해서
            days += totalDay[i];
        }

        days += b - 1;  // 찾으려는 요일의 일을 더함 1일부터 시작이므로 -1

        int index = days % 7;   // 1월 1일부터 찾으려는 요일의 날짜까지의 총 일수에서 7을 나눔

        return day[index];  // 요일 찾아서 리턴
    }
}

File: yongkeun/6week/2016년.js
function solution(a, b) {
  //1월 31일
  //2월 29일
  //3월 31일
  //4월 30일
  //5월 31일
  //6월 30일
  //7월 31일
  //8월 31일
  //9월 30일
  //10월 31일
  //11월 30일
  //12월 31일
  const day = ["FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"]; //1월1일은 금요일이므로 FRI부터 배열에 담기 시작
  var answer = 0;
  for (let i = 1; i < a; i++) {
    //5월이면 1,2,3,4를 반복
    //switch case문을 이용해 무슨 달을 지났는지 check하고 그 해당 달의 일수만큼 더함
    switch (i) {
      case 3:
      case 5:
      case 7:
      case 8:
      case 10:
      case 12:
        answer += 31;
        break;
      case 1: //1월은 31일까지 있지만, 1월1일부터 시작하므로 30일만 더하면 됨
      case 4:
      case 6:
      case 9:
      case 11:
        answer += 30;
        break;
      case 2:
        answer += 29;
        break;
    }
  }
  if (a === 1) {
    //만약에 1월이라면 1월1일부터 시작하므로 -1을 뺌
    b -= 1;
  }
  answer += b; //이제 월에서 며칠이 지났는지 계산한 값과, 현재 일(b)를 더함
  return day[answer % 7]; //값을 7로 나눈 나머지 값이 day의 index임
}
